core/common/buffer.cpp

```diff
@@ -139,6 +139,10 @@ namespace kagome::common {
     return put(buf.toVector());
   }
 
+  Buffer &Buffer::putBuffer(Buffer &&buf) {
```

Comment:
it makes no sense, the content is copied anyway, and then the object is destroyed